config/logging.yaml

```diff
@@ -33,8 +33,18 @@
 #   includes filename and line number of the log message)
 #   values: true, false
 #
+# :layout:
+#   Formatter, one of 'pattern', 'multiline_pattern' or 'json'. The multiline pattern prepends
+#   '|' characters on all additional lines for better readability.
+#
+# :items:
```

Comment:
`s/items/json_items`, otherwise, it's not clear what this value is being used for.

Reply:
Yeah makes sense.